tests/TestCase/Core/Configure/Engine/JsonConfigTest.php

```diff
@@ -96,7 +96,7 @@ public function testReadWithNonExistentFile()
      * Test reading an empty file.
      *
      * @expectedException \Cake\Core\Exception\Exception
-     * @expcetedExceptionMessage Decoding JSON config file "empty.json" did not return any array
+     * @expectedExceptionMessage Decoding JSON config file "empty.json" did not return an array
```

Comment:
If I replace the error message my 5.6 unit tests fail because the exception message is the same as above.
 
I've tried adding multiple @expectedExceptionMessage's without any luck.

My thoughts are:
1. Use @expectedExceptionMessageRegExp to come up with a regular expression that matches on both and is detailed enough not to catch unwanted exceptions.
2. Remove the expected message and reply on expectedException.

I'm still learning and unsure. What are your thoughts?